new/PRS14371Swaynn.xml

```diff
@@ -0,0 +1,60 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14371Swaynn" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>H. G. C. Swaynn</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-03-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1"><forename>H.</forename> <forename>G.</forename> 
+                        <forename>C.</forename> <surname>Swaynn</surname></persName>
```

Comment:
Swayne

Reply:
https://www.abhilekh-patal.in/jspui/handle/123456789/4159727?searchWord=&backquery=[query=&filter_field_1=subjectCollection&filter_type_1=equals&filter_value_1=Digitized+Public+Records&filter_field_2=keywords&filter_type_2=equals&filter_value_2=Bombay&originalquery=&sort_by=dc.date.accessioned_dt&order=desc&rpp=20&etal=0&start=40]

https://en.wikipedia.org/wiki/Harald_George_Carlos_Swayne

Reply:
It is Capt. Swaynn in the catalogue description. It is maybe a misreading by Strelcyn. I think, it is reasonable to change to Swayne. I will notify the difference in the manuscript record in a note.

![grafik](https://github.com/user-attachments/assets/03e6574a-f035-4c0c-bfea-640389fdae16)